api/apps/geoprocessing/src/modules/features/features.service.ts

```diff
@@ -44,10 +45,12 @@ export class FeatureService {
    * @todo generate the custom queries using query builder and the entity data.
    * @todo move the string to int transformation to the AdminAreaLevelFilters class
    */
+
   buildFeaturesWhereQuery(id: string, bbox?: BBox): string {
     let whereQuery = `feature_id = '${id}'`;
+
     if (bbox) {
-      whereQuery += `AND the_geom && ST_MakeEnvelope(${bbox}, 4326)`;
+      whereQuery += `AND st_intersects(ST_MakeEnvelope(${nominatim2bbox(bbox)}, 4326), the_geom)`;
```

Comment:
`nominatim2bbox(bbox)` suggests bbox is a `nominatim` which is converted to bbox
it's a bit confusing, because then bbox is not really a bbox

Reply:
indeed both are bbox, the difference is that they are using 2 different standards, for know i will leave it like that